backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/FRBR.java

```diff
@@ -0,0 +1,76 @@
+package de.bund.digitalservice.ris.norms.domain.entity;
+
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import de.bund.digitalservice.ris.norms.utils.exceptions.MandatoryNodeNotFound;
+import java.time.LocalDate;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+
+/**
+ * Class representing a generalization of FRBR elements. It cannot be instantiated itself because it
+ * needs a custom implementation of a level.
+ */
+@Getter
+@AllArgsConstructor
+@SuperBuilder(toBuilder = true)
+public abstract class FRBR {
+  private final Node node;
+  private final String normEli;
+  private final String nodeName;
+
+  /**
+   * Returns the Eli as {@link String} from the FRBRThis of the specific FRBR level.
+   *
+   * @return An Eli
+   */
+  public String getEli() {
+    final String xpath = "./FRBRthis/@value";
+    return NodeParser.getValueFromExpression(xpath, node)
+        .orElseThrow(() -> new MandatoryNodeNotFound(xpath, this.nodeName, this.normEli));
+  }
+
+  /**
+   * Updates the Eli of a Norm
+   *
+   * @param eli - the new ELI
+   */
+  public void setEli(final String eli) {
+    final String xpath = "./FRBRthis/@value";
+    NodeParser.getNodeFromExpression("./FRBRthis", node)
+        .orElseThrow(() -> new MandatoryNodeNotFound(xpath, this.nodeName, this.normEli))
+        .getAttributes()
+        .getNamedItem("value")
+        .setNodeValue(eli);
+  }
+
+  /**
+   * Returns a FBRDate as {@link LocalDate} from the FRBR level.
+   *
+   * @return The FBRDate
+   */
+  public LocalDate getFBRDate() {
+    final String xpath = "./FRBRdate/@date";
+    return NodeParser.getValueFromExpression(xpath, node)
+        .map(LocalDate::parse)
+        .orElseThrow(() -> new MandatoryNodeNotFound(xpath, this.nodeName, this.normEli));
+  }
+
+  /**
+   * Updates both the date and the name attibutes of FBRDate
+   *
+   * @param date - the new date
+   * @param name - the new name
+   */
+  public void setFBRDate(final String date, final String name) {
```

Comment:
if the `get` returns a date I think the `set` should also get a date (this could also remove the need to format the date in the `LoadZf0UseCase`)

Reply:
I would prefer to return a String in the getFRBRDate than to pass a LocalDate since we don't have any requirements now for using a LocalDate like compare them or something like that. For now we just get a date from the XML, in the ISO string format and set it in another FRBRDate. What do you think?

Reply:
Works for me as well, as long as getter and setter work with the same type :)